.template-helpers/variant.md

```diff
@@ -0,0 +1,7 @@
+# Image Variants
+
+The `%%REPO%%` images come in three flavors, each designed for a specific use case.
+
+`<version>`:
+
+This is the defacto image. If you are unsure about what your needs are, you probably want to use this one. It is designed to be used both as a throw away container (mount your source code and start the container to start your app), as well as the base to build other images off of. This tag is based off of [`buildpack-deps`](https://registry.hub.docker.com/_/buildpack-deps/). `buildpack-deps` is designed for the average user of docker who has many images on thier system. It, by design, has a large number of extremely common debian packages. This reduces the number of packages that images that derive from it need to install, thus reducing the total size of all of your docker images on your system.
```

Comment:
"all of your docker images on your system" is a bit too much "your"

How about just "... thus reducing the overall size of all images on your system" ?